catalyst-gateway/tests/schemathesis_tests/Earthfile

```diff
@@ -114,8 +119,8 @@ nightly-test-fuzzer-api:
         --service event-db \
         --service cat-gateway \
         --allow-privileged
-
-        RUN --no-cache docker run --net=host --name=st schemathesis:latest || echo fail > fail; \
+        # Need to wait until cat-gateway will successfully connect to the scylladb
+        RUN --no-cache sleep 120; docker run --net=host --name=st schemathesis:latest || echo fail > fail; \
```

Comment:
I dont think it is a good idea to wait for a fixed amount of time, maybe would be bettere to use a [hook](https://schemathesis.readthedocs.io/en/stable/extending.html#after-load-schema) to ping the `/ready` endpoint before starting the test?

Reply:
Had this similar issue, work around by checking both event-db and cat-gateway started and service healthy
```
cat-gateway-is-running:
    image: alpine:3.20.3
    depends_on:
      event-db:
        condition: service_healthy
      cat-gateway:
        condition: service_started
 ```

Reply:
In this case I need to wait until cat-gateway will connect to the scylladb, unfortunately `service_started` will not help in this scenario.
`/ready` endpoint will not help  as well, at this moment it is not doing much. its not checking connections to the databases.

Reply:
If you can make the scylla-db work in another container that would solve the issue, otherwhise can you please open a ticket so we can revisit this sleep approch when the /ready endpoint will be working? Thanks!